app/screens/Settings/SettingsScreen.tsx

```diff
@@ -60,9 +66,12 @@ const SettingsScreen = (props) => {
 
   return (
     <ScrollView style={styles.container}>
-      {rowItems.map((item, index) => (
-        <SettingsRow key={index} onPress={item.onPress} title={item.title} />
-      ))}
+      {rowItems.map((item, index) => {
+        if (item.os && item.os !== Platform.OS) return null;
```

Comment:
```
if(...) {
   return null;
}
```

Reply:
I wonder why the eslint is not active for this. in my personal projects it is activated by default preventing me to do such a thing.